styleguide/source/_patterns/02-organisms/03-sections/07-simple-step-form/simple-step-form.twig

```diff
@@ -0,0 +1,32 @@
+{#
+ # @file simple-step-form.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<form class="simple-step-form" method="get" action="" novalidate="novalidate">
+  <fieldset>
+    <p class="h2">To reach us by email, please fill out the following:</p>
+    {% include 'molecules-input-text' %}
+    {% include 'molecules-input-telephone' %}
+    {% include 'molecules-input-select' %}
+    {% include 'molecules-input-email' %}
+    {% include 'molecules-input-radio' %}
+    {% include 'molecules-input-month' %}
+    {% include 'molecules-input-number' %}
```

Comment:
@aCyborg I guess I'm not too concerned about the DOB field year range because this is an HTML5 component field that allows for a UI tool. I did add a min max to the YYYY field though.